HW4.py

```diff
@@ -0,0 +1,360 @@
+import os
+import sys
+
+import numpy as np
+import pandas as pd
+import scipy.stats as ss
```

Comment:
The `numpy` module was giving me errors at first (my pip compile chose to use 1.8, I think).  Based on [this](https://github.com/scipy/scipy/issues/14541) thread, this seems to be due to an interface change.  Changing to 1.9 seemed to solve this.